### DISCUSSION TITLE
wrong links to constructors in pandoc lua filters web page; clarify that TableBody has no constructor

### DISCUSSION THREAD

**Discussion by massifrg** [jgm/pandoc#10435] [General]
I found a problem in the web page about [Lua filters](https://pandoc.org/lua-filters.html).

The links to constructors don't work, because they are made lowercase.
For example, in the [Pandoc type](https://pandoc.org/lua-filters.html#type-pandoc) section, there's a link to the `pandoc constructor`, whose `href` is pointing to:

`https://pandoc.org/lua-filters.html#pandoc.pandoc`

instead of

`https://pandoc.org/lua-filters.html#pandoc.Pandoc`

so the link doesn't work in the browser.

It would be also useful underlining, in the docs, that TableBody has no constructor and you instantiate one simply with a table having the `attr` (`Attr`), `body` (list of `Row` items), `head` (list of `Row` items) and `row_head_columns` (integer) fields.

(I apologize for not making a PR fixing these issues)

**Comment by wlupton** [comment]
I've been meaning to say something similar! Also note that the headers have apparently inappropriate "extension" info (see screenshot).

![Screenshot 2024-12-04 at 17 15 59](https://github.com/user-attachments/assets/d22a6641-d452-41e1-8703-21db5b68f381)

**Reply by jgm** [reply]
@tarleb will know how to fix these things.

**Reply by jgm** [reply]
I fixed the extension support links.

**Reply by tarleb** [reply]
Constructor links are fixed via 8c96c6e7768fbf4af8ebce1126400b7655de7a4d.

**Reply by jgm** [reply]
I pushed this to the website.
